tests/cypress/page-object/base.page.ts

```diff
@@ -44,4 +50,38 @@ export class BasePage {
                 .then(cy.wrap)
         )
     }
+
+    getIframeBodySelector(
+        selector: string,
+        options?: Partial<Cypress.Loggable & Cypress.Timeoutable & Cypress.Withinable & Cypress.Shadow>,
+    ): Cypress.Chainable {
+        return cy.get(selector, options).its('0.contentDocument.body').should('not.be.empty').then(cy.wrap)
+    }
+
+    getIframeElement(iframeSelector: string, elementSelector: string, timeout = 60000): Cypress.Chainable {
+        return cy
+            .get(iframeSelector, { timeout: timeout })
+            .should(($iframe) => {
+                expect($iframe.contents().find(elementSelector)).to.exist
+            })
+            .then(($iframe) => {
+                return cy.wrap($iframe.contents().find(elementSelector))
+            })
+    }
+
+    containIframeElement(
+        iframeSelector: string,
+        elementSelector: string,
+        text: string,
+        timeout = 60000,
+    ): Cypress.Chainable {
+        return cy
+            .get(iframeSelector, { timeout: timeout })
+            .should(($iframe) => {
+                expect($iframe.contents().find(`${elementSelector}:contains(${text})`)).to.exist
```

Comment:
instead of `.find(`${elementSelector}:contains(${text})`))` , the following should work `.contains(elementSelector, text)`